src/main/php/lang/ast/Emitter.class.php

```diff
@@ -331,7 +331,7 @@ protected function emitClosure($closure) {
   }
 
   protected function emitLambda($lambda) {
-    $this->out->write('function'); 
+    $this->out->write('@function');
```

Comment:
This is not great - we have to suppress errors coming from a `use` list generated without actually testing the local scope for the variables it includes. In case the lambda contains variables which don't exist in the outer scope, they should not be added to the `use` list!